src/deluge/gui/views/automation_instrument_clip_view.h

```diff
@@ -60,9 +60,8 @@ class AutomationInstrumentClipView final : public ClipView, public InstrumentCli
 	                    uint8_t occupancyMask[][kDisplayWidth + kSideBarWidth], bool drawUndefinedArea = true);
 	bool renderSidebar(uint32_t whichRows, uint8_t image[][kDisplayWidth + kSideBarWidth][3],
 	                   uint8_t occupancyMask[][kDisplayWidth + kSideBarWidth]);
+	void renderDisplay(int32_t knobPos = kNoSelection);
 	void displayAutomation();
```

Comment:
replaced displayParameterName by new renderDisplay function
eliminated needed for setDisplayParameterNameTimer through refactoring of display code